components/ws-manager-bridge/src/redis/client.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { inject, injectable, postConstruct } from "inversify";
+import { Redis } from "ioredis";
+import { Configuration } from "../config";
+
+@injectable()
+export class RedisClient {
```

Comment:
I've copied this from `server`. How do we want to share this `Client` implementation between `server` and `ws-man-bridge`?

It's worth noting that this current implementation won't allow a given component to be both a subscriber, and publisher in pubsub. To have that, we'd need to create two distinct instances of the `client`. However, we don't yet have this use-case.

Reply:
I'll keep it here as-is for now